src/main/java/io/nats/service/ServiceUtil.java

```diff
@@ -0,0 +1,73 @@
+// Copyright 2022 The NATS Authors
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at:
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package io.nats.service;
+
+import io.nats.service.api.EndpointStats;
+
+import java.time.Duration;
+import java.util.Comparator;
+
+import static io.nats.client.support.Validator.nullOrEmpty;
+
+public abstract class ServiceUtil {
+
+    static final String PING = "PING";
+    static final String INFO = "INFO";
+    static final String SCHEMA = "SCHEMA";
+    static final String STATS = "STATS";
+    static final String SRV_PREFIX = "$SRV.";
```

Comment:
This prefix [needs to be overridable](https://github.com/nats-io/nats-architecture-and-design/blob/4a68e204c3bab514e9818ca113c173db8bd2408c/adr/ADR-33.md#discovery-and-status) just like the `$JS.` is overridable.

It's okay if that's not part of the Beta yet, just want to check.

Reply:
correct, it will need to be overridden, but how has not been determined yet. There are currently 2 ways, one for JetStream api and the other for kv/object apis. I've asked, and as you might have noticed it's absent from the adr. Maybe I should just name it to be default prefix?